src/pytorch_lightning/utilities/meta.py

```diff
@@ -144,7 +146,9 @@ def is_meta_init() -> bool:
 
 else:
 
-    def init_meta(*_, **__):
+    def init_meta(
+        module_fn: Callable[..., Module], *args: Any, **kwargs: Any
+    ) -> Union[Module, MisconfigurationException]:
         if not _TORCH_GREATER_EQUAL_1_10:
             return MisconfigurationException("`init_meta` is supported from PyTorch 1.10.0")
```

Comment:
wondering, why it returns an Exception instead of raising it?

Reply:
Looks like a bug

Reply:
I modified definition line of `init_meta` function because `mypy` shows error when conditional function do not have same identical signature.
And original code returns `MisconfigurationException`, so I followed to return error without additional modify